docs/source/install-linux.rst

```diff
@@ -23,7 +23,7 @@ Ubuntu 14.04
 
 1. Type the following into your Terminal window to install Python 3.5::
 
-    sudo add-apt-repository ppa:fkrull/deadsnakes
```

Comment:
Per the info provided by [deadsnakes](https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa), it should be `sudo add-apt-repository ppa:deadsnakes/ppa`

Reply:
heh, I'm the maintainer.  `add-apt-repository ppa:deadsnakes` is shorthand for `ppa:deadsnakes/ppa` btw :)